pkg/proxy/healthcheck/healthcheck_test.go

```diff
@@ -127,8 +127,10 @@ type healthzPayload struct {
 func TestServer(t *testing.T) {
 	listener := newFakeListener()
 	httpFactory := newFakeHTTPServerFactory()
+	fakeClock := testingclock.NewFakeClock(time.Now())
```

Comment:
We have to add new testcases with the new combinations that we are adding

Reply:
Will do